samples/cpp/CMakeLists.txt

```diff
@@ -115,6 +115,8 @@ endif()
 if(TARGET gflags)
     set(GFLAGS_TARGET gflags)
 elseif(gflags_required)
+    # set CMAKE_POLICY_VERSION_MINIMUM because gflags requires cmake 3.0.2 or higher
+    set(CMAKE_POLICY_VERSION_MINIMUM 3.5)
```

Comment:
Why is there cmake 3.0.2 or higher in the comment but CMAKE_POLICY_VERSION_MINIMUM is 3.5?